### ISSUE TITLE
Ticker: 0097.KL missing balance_sheet

### ISSUE THREAD

**Issue by mrkgoh** [ranaroussi/yfinance#1001]
It seems that yfinance doesn't work for Malaysia stocks?

Here are the code:
```
import yfinance as yf
code2 = '0097.kl'  # Malaysia Stock Name Vitrox
m = yf.Ticker(code2)
m.balance_sheet
>>>
	Open	High	Low	Close	Adj Close	Volume
Date						
```

As you can see from the print screen the data exists on Yahoo Finance page:
https://finance.yahoo.com/quote/0097.KL/balance-sheet?p=0097.KL

![Capture](https://user-images.githubusercontent.com/30623155/167667163-711f3798-84f9-4878-b997-e237bb5f8f52.JPG)

**Comment by mrkgoh** [comment]
Here is my attempt at debugging:
1. First I try to inspect the html I am getting. Not from https://finance.yahoo.com/quote/0097.KL/financial/ but from 
```
import requests as _requests
import json as _json
code = '0097.kl'

_SCRAPE_URL_ = 'https://finance.yahoo.com/quote'
ticker_url = "{}/{}".format(_SCRAPE_URL_, code)

head = {
    'accept': '*/*',
    'accept-encoding': 'gzip, deflate',
    'accept-language': 'en-US,en;q=0.9',
    'referer': 'https://s.yimg.com/rq/darla/4-10-1/html/r-sf.html',
    'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.41 Safari/537.36',
    
}

s = _requests.session()  # Used session to hopefully load caches
s.get(url=ticker_url, headers=head)
html = s.get(url=ticker_url + f'/balance-sheet?p={code}', headers=head).text
```
html output as attached
[html.txt](https://github.com/ranaroussi/yfinance/files/8667621/html.txt)

**Comment by ValueRaider** [comment]
Correct financials are now available in pre-release version 0.2.0rc4.
